package/efs/rancher-efs

```diff
@@ -214,14 +220,21 @@ mountdest() {
 
     get_meta_data
 
-    efsDNSName=${EC2_AVAIL_ZONE}.${OPTS[fsid]}.efs.${EC2_REGION}.amazonaws.com
+    local efsMountIP="${OPTS[mountTargetIp]}"
+    # Amazon takes a while to make the mount target resolvable via DNS, so...
+    while [ "$efsMountIP" == "" ]; do
```

Comment:
Why did you switch to an IP? Is it just for because you want to wait until it resolves? If that is the reason, could we just test the return code of dig?

Do we gain anything by using the DNS name? In theory we do, but in practice, it looks like it is an explicit action only in the UI to change a mount target's IP, so it probably is not common. But, if someone _did_ do that, they'd be pretty much screwed. So, I think DNS name is better, but can be convinced otherwise if you have another reason that I didn't think of.